rust/parser/typeql.pest

```diff
@@ -354,37 +354,42 @@ DATETIME_ = @{ DATE_FRAGMENT_ ~ "T" ~ TIME_ ~ WB }
 VAR_ = ${ VAR_CONCEPT_ | VAR_VALUE_ }
 VAR_CONCEPT_ = @{ VAR_CONCEPT_ANONYMOUS_ | VAR_CONCEPT_NAMED_ }
 VAR_CONCEPT_ANONYMOUS_ = @{ "$_" ~ WB }
-VAR_CONCEPT_NAMED_ = @{ "$" ~ IDENTIFIER }
-VAR_VALUE_ = @{ "?" ~ IDENTIFIER }
+VAR_CONCEPT_NAMED_ = @{ "$" ~ IDENTIFIER_VAR_H_ ~ IDENTIFIER_VAR_T_*  ~ WB }
+VAR_VALUE_ = @{ "?" ~ (IDENTIFIER_VAR_H_ ~ IDENTIFIER_VAR_T_* ~ WB) }
 IID_ = @{ "0x" ~ ASCII_HEX_DIGIT+ ~ WB }
-LABEL_ = @{ IDENTIFIER }
-LABEL_SCOPED_ = @{ IDENTIFIER ~ ":" ~ IDENTIFIER }
-IDENTIFIER = @{ TYPE_CHAR_H_ ~ TYPE_CHAR_T_* ~ WB }
+LABEL_ = @{ (IDENTIFIER_LABEL_H_ ~ IDENTIFIER_LABEL_T_* ~ WB) }
+LABEL_SCOPED_ = @{ (IDENTIFIER_LABEL_H_ ~ IDENTIFIER_LABEL_T_*) ~ ":" ~ (IDENTIFIER_LABEL_H_ ~ IDENTIFIER_LABEL_T_*)  ~ WB }
```

Comment:
We should probably have an `IDENTIFIER_LABEL_` rule.

Reply:
Although the ANTLR grammar reuses `LABEL_` instead. If we do that, `IDENTIFIER_VAR_` might have to go as well. I'd rather we introduce both in ANTLR though.

Reply:
I just canceled them all, so we don't end up with confusion re. `~wb`